samples/Riok.Mapperly.Sample/Riok.Mapperly.Sample.csproj

```diff
@@ -2,8 +2,6 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>net7.0</TargetFramework>
```

Comment:
I would leave this in here to make it easier for others to copy/view this project

Reply:
The csproj is not easy copyable at all due to the Mapperly references beeing `ProjectReference` instead of `PackageReference`. Due to that fact I think we should keep the project file as maintainable and as simple as possible for us.